apiary.apib

```diff
@@ -3281,6 +3281,302 @@ Update the current timer. Only possible if there is a timer running.
+                + duration: 3600 (number, required) - In seconds
+                + description (string, optional)
+        + materials (array, optional)
+            + (object)
+                + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
+                + quantity: 3 (number, required)
+        + custom_fields (array[CustomField], optional)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + type: `workOrder` (string)
+            + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
+
+### workOrders.update [POST /workOrders.update]
+
+Update a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
+        + customer (object, optional)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + starts_at: `2017-05-26T10:01:49+00:00` (string, optional)
+        + description: `Pipework intervention` (string, optional, nullable)
+        + department_id: `cef01135-7e51-4f6f-a6eb-6e5e5a885ac7` (string, optional, nullable)
+        + layout_id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string, optional, nullable)
+        + milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional, nullable)
+        + event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional, nullable)
+        + location (Address, optional, nullable)
+        + distance (Distance, optional, nullable)
+        + time (array, optional)
+            + (object)
+                + user_id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string, required)
+                + started_at: `2017-04-26T10:01:49+00:00` (string, required)
+                + duration: 3600 (number, required) - In seconds
+                + description (string, optional)
+        + materials (array, optional)
+            + (object)
+                + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
+                + quantity: 3 (number, required)
+        + custom_fields (array[CustomField], optional)
+
++ Response 204
+
+### workOrders.list [GET /workOrders.list]
+
+Get a list of work orders.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + customer (object, optional)
+                + type (enum[string], required)
+                    + Members
+                        + contact
+                        + company
+                + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+            + state: `draft`, `finalized` (enum[string], optional)
+                + (enum[string])
+                    + draft
+                    + finalized
+                    + sent
```

Comment:
Q: Isn't this part incorrect? Renders like this:
![Screenshot 2019-05-21 14 02 21](https://user-images.githubusercontent.com/9056632/58094475-1445a280-7bd1-11e9-89fd-c171200f8837.jpg)

Reply:
You can check the https://github.com/teamleadercrm/api/compare/work-orders branch.
That one has the approved change for work orders.
This PR will never get merged by the way.

Reply:
But now that you mention this. 
The state is OK in the branch that I mentioned.

But the `sort.field` and `sort.order` have the same thing. 
But I can't seem to be able to use the correct syntax to fix it 😞

Reply:
That's really hard to find IMO... Isn't there a better way to look at this code? There's no PR for that branch open yet right now?

Reply:
Not yet. But you can checkout the branch and make a preview apiary.html file.

Reply:
How Apiary renders is it admittedly weird. Not sure how we can fix this but it's an issue for multiple endpoints.

Since we revived an internal API endpoints repo we could add them there in the future and let the docs be rendered separately. If we have spare time we could/might move the ones in the integration branch already.